docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -484,6 +484,27 @@ The monitor will now:
 2. Watch for large dex swaps.
 3. Send notifications via Slack when large swaps occur.
 
+**Running monitor against a specific block**
+
+You can run the monitor against a specific block number for testing or debugging purposes. This is useful when you want to verify how the monitor behaves.
+
+[source,bash]
+----
+cargo run -- --monitorName "Large Transfer of USDC Token" --network=ethereum_mainnet --block=12345678
```

Comment:
Would it makes sense to reference the path to the json file of the monitor here instead? This approach would require "monitorName" to be unique across all your monitors and could make it a bit tricky to maintain unique names across all your monitors. With a reference to the file, we could just load the monitor and test against the latest block number of all networks defined in there, OR if a specific `--block` flag is defined, we also require you to define the `--network` flag.
Additionally, we would need some validation in place that the network is defined for that monitor.

Just thinking out loud here, would that make sense?